manim/animation/creation.py

```diff
@@ -397,6 +398,62 @@ def __init__(
         )
 
 
+class SpiralIn(Animation):
+    r"""Create the Mobject with sub-Mobjects flying in on spiral trajectories.
+
+    Parameters
+    ----------
+    shapes
+        The MObject on which to be operated.
+
+    scale_factor
+        The factor used for scaling the effect.
+
+    Examples
+    --------
+    .. manim :: SpiralInExample
+
+        class SpiralInExample(Scene):
+            def construct(self):
+                pi = MathTex(r"\pi").scale(7)
+                pi.shift(2.25 * LEFT + 1.5 * UP)
+                circle = Circle(color=GREEN_C, fill_opacity=1).shift(LEFT)
+                square = Square(color=BLUE_D, fill_opacity=1).shift(UP)
+                shapes = VGroup(pi, circle, square)
+                self.play(SpiralIn(shapes))
+    """
+
+    def __init__(
+        self,
+        shapes: Mobject,
+        scale_factor: float = 1,
```

Comment:
Why not set the default value of `scale_factor` to 8 here, rather than multiplying it later by 8? Currently, users would not be able to determine what exactly this `scale_factor` corresponds to without referring to the source code.

Reply:
Yeah, so a lot of this code was me just essentially copy and pasting the "hard coded" animation for the manim logo. scale_factor, for instance, is not something that I added here. I tried to change the code as minimally as possible while still allowing it to be generalizable to arbitrary Mobjects. I'm okay changing it, but I wonder if the original author of the animated manim logo had a particular reason for doing things this way. Maybe I'm overthinking it. As far as the scale factor goes, your guess is as good as mine as to what it really means here.

Reply:
Done!